### PR TITLE
Fix barchart

### PR DESCRIPTION
**Pull request by starmon00**
This PR changed the linechart into a barchart in the front end webpage.

Related issues: #105 

@JavierJia

### CODE REVIEW

File: neo/public/javascripts/timeseries/controllers.js [hunk 1]
```diff
@@ -74,14 +77,15 @@ angular.module('cloudberry.timeseries', ['cloudberry.common'])
               .html(minDate.getFullYear()+"-"+(minDate.getMonth()+1)+"-"+minDate.getDate());
 
             timeSeries
-              .renderArea(true)
```
Comment (JavierJia) [inline neo/public/javascripts/timeseries/controllers.js]: just curious, why should we remove this line? though I'm not sure what it is doing?

**Comment by JavierJia** [comment]
👍  looks good.

**Comment by JavierJia** [comment]
@starmon00 I think the gap still have some problem, I pushed it online: http://cloudberry.ics.uci.edu/
You can see that the bar is too narrow yet the gap is large, I think the gap should be narrow.

**Comment by starmon00** [comment]
@JavierJia ok let me add a new line of code that modifies the gap.

**Pull request merged**